save-core/src/commonMain/kotlin/org/cqfn/save/core/plugin/PluginException.kt

```diff
@@ -0,0 +1,6 @@
+package org.cqfn.save.core.plugin
+
+/**
+ * An [Exception] that can be thrown when parsing a file
+ */
+class PluginException(message: String) : Exception(message)
```

Comment:
We should have not only a general `PluginException`, but a dedicated exception for malformed test resources